client/tests/westend.test.ts

```diff
@@ -0,0 +1,319 @@
+import {
```

Comment:
Lots of duplication with `rococo.test.ts`, does it make sense to merge them into a single, parametrized set of tests?

Reply:
You are right. I did it quite in a hurry, so let me fix that.